### PR TITLE
Feature/results page layout

### PR DESCRIPTION
**Pull request by tsiananel**
image

### CODE REVIEW

File: src/app/models/result/result.component.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <p>
+    <b>Listening</b> Test Module score is <b> {{ currentAuditionResult }} </b>
+  </p>
+  <p>
+    <b>Writing</b> and <b>Speaking</b> Test modules are sent for checking and the results will be
+    sent by email.
+  </p>
+
+  <button class="btn" mat-raised-button color="primary" [routerLink]="'/home'">To Home</button>
+</div>
```
Comment (shuliakleonid) [inline src/app/models/result/result.component.html:14]: use enum Route

File: src/app/services/store/test-store.service.ts [hunk 2]
```diff
@@ -149,6 +147,7 @@ export class TestStoreService {
   timer(counter: number, interval: number, func: () => void): Subscription {
     const obs = timer(0, interval).pipe(
       takeWhile(() => counter > 0),
+      // eslint-disable-next-line no-param-reassign
       tap(() => (counter -= 1)),
     );
```
Comment (shuliakleonid) [inline src/app/services/store/test-store.service.ts:152]: it possible rewrite

**Review by shuliakleonid — CHANGES_REQUESTED** [review]

File: src/app/models/common-test/common-test.component.html [hunk 3]
```diff
@@ -6,44 +6,24 @@
       [questions]="grammar$ | async"
       (answersGrammar)="onGrammarSubmit($event)"
     ></app-grammar-test>
-    <app-proceed-buttons
```
Comment (RomPavel) [inline src/app/models/common-test/common-test.component.html:9]: Did you discuss this change with BA?

Comment (tsiananel) [reply src/app/models/common-test/common-test.component.html:9]: Yes, it's for delete(

**Review by shuliakleonid — APPROVED** [review]

**Pull request merged**